components/playlist/browser/resources/components/app.tsx

```diff
@@ -120,7 +120,7 @@ export class PlaylistPage extends React.Component<Props, State> {
                   name={item.name}
                   cached={item.cached}
                   thumbnailUrl={this.getImgSrc(item)}
-                  onClick={this.onClickItem.bind(this)}
+                  onClick={(...args) => this.onClickItem(...args)}
```

Comment:
If it just needs one variable you can also forward it the normal way (but this is 100% fine)

e => this.onClickItem(e)